TransferMethod/Sources/SelectTransferMethodTypeController.swift

```diff
@@ -117,7 +134,17 @@ extension SelectTransferMethodTypeController {
     }
 
     override public func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        return Theme.Cell.largeHeight
+        return rowHeightConsideringSizeCategory(for: indexPath)
+    }
+}
+
+extension SelectTransferMethodTypeController: ContentSizeCategoryAdjustable {
+    public func rowHeightConsideringSizeCategory(for indexPath: IndexPath) -> CGFloat {
```

Comment:
Can this method be added in extension of UIViewController instead of adding inside individual controllers. Pass indexPath: IndexPath and default height as parameters.

Reply:
Updated